Nodejs/Product/Nodejs/SharedProject/CommonProjectNode.cs

```diff
@@ -904,7 +830,7 @@ private void OnIdle(object sender, ComponentManagerEventArgs e)
                     {
                         // we have more file merges to process, do this
                         // before reflecting any other pending updates...
-                        if (!merger.ContinueMerge(this.ParentHierarchy != null))
+                        if (!(await merger.ContinueMergeAsync(this.ParentHierarchy != null)))
```

Comment:
>if [](start = 24, length = 2)

The caller above used a while loop.  Is the difference important?

Reply:
Yes, here we want to check if VS is still idle between each folder we process.